### ISSUE TITLE
(feature request) automatic Number -> Integer conversion for non-decimal values imported from JSON

### ISSUE THREAD

**Issue by wyatt8740** [gmariani/minerva#76]
Importing a JSON file does not let Minerva know what data type is preferred for integer values. It appears to assign them all to the "Number" data type by default.

I would suggest either some kind of "meta data" included in the exported JSON which specifies data types, or a way to have the program automatically decide on importation.

Additionally, some kind of scripting interface so that this kind of thing could be automated by users would be nice, but it appears that the entire program is encapsulated in an anonymous function right now.

**Comment by gmariani** [comment]
This would be very far down the road if I did do this. Minerva is made to service a dying technology and it's a side-project. To build a scripting API to automate this is a bit beyond the scope of what minerva was made to do. I can look into passing data type metadata, but again, a bit back burner, atleast for now.

**Comment by gmariani** [comment]
Yeah it's been almost two years and what i wrote then is even more true now and i haven't had time to get to this as it is. going to drop this.

**Comment by wyatt8740** [comment]
Fair enough; I understand your reasoning.

If I were to find some way you deemed elegant and did a pull request, would it have a shot at getting accepted?

**Comment by gmariani** [comment]
Sure, I'd be open to that.
